br/pkg/streamhelper/flush_subscriber.go

```diff
@@ -125,14 +125,15 @@ func (f *FlushSubscriber) Drop() {
 // HandleErrors execute the handlers over all pending errors.
 // Note that the handler may cannot handle the pending errors, at that time,
 // you can fetch the errors via `PendingErrors` call.
-func (f *FlushSubscriber) HandleErrors(ctx context.Context) {
+func (f *FlushSubscriber) HandleErrors() {
 	for id, sub := range f.subscriptions {
 		err := sub.loadError()
 		if err != nil {
 			retry := f.canBeRetried(err)
 			log.Warn("Meet error.", zap.String("category", "log backup flush subscriber"),
 				logutil.ShortError(err), zap.Bool("can-retry?", retry), zap.Uint64("store", id))
 			if retry {
+				log.Info("retry connecting to store to add subscription")
```

Comment:
It seems this log can be derived from `zap.Bool("can-retry?", retry)` in the previous log?

Reply:
how about removing l134, and track the record id to l136?